doc/nrf/samples/benchmarks.rst

```diff
@@ -0,0 +1,11 @@
+.. _benchmarks_samples:
+
+Benchmarks samples
```

Comment:
to be visible in the doctree this also needs to be included in the 
doc/nrf/samples.rst (the paths there are listed alphabetically)

so that would be a new line under amazon_sidewalk:
samples/benchmarks